foundationdb/src/future.rs

```diff
@@ -251,6 +253,120 @@ impl AsRef<CStr> for FdbAddress {
     }
 }
 
+/// An slice of keys owned by a FoundationDB future
+#[cfg_api_versions(min = 700)]
```

Comment:
maybe put it in a module and then export it to reduce cfg_api_versions usages.

#[cfg_api_versions(min = 700)]
mod fdb700 {
    ...
}
pub use fdb700::FdbKeys

Reply:
Thanks @Speedy37, I applied your suggestion :+1: